### PR TITLE
added gorilla/mux-redis sample app

### PR DESCRIPTION
**Pull request by developer-diganta**
# Pull Request Template

## Description

I have added the sample gorilla/mux-redis app to go's quickstart guide.

## Additional Context (Please include any Screenshots/gifs if relevant)

image

## Checklist:

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
- [x] I have made corresponding changes to the documentation.
- [ ] I have added corresponding tests.
- [ ] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Comment by nehagup** [comment]
@developer-diganta sign DCO please

**Comment by developer-diganta** [comment]
Done

File: docs/go/quickstart/quickstart.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ tags:
...
+- [URL Shortener with Echo-SQL](/docs/go/quickstart/echo-sql) - A sample URL Shortener app that responds with a shortened URL of the URL sent as a request. The application tests Keploy integration capabilities using [Echo](https://echo.labstack.com/) and [PostgreSQL](https://www.postgresql.org/).
 - [URL Shortener with Gin-Mongo](/docs/go/quickstart/gin-mongo) - A sample application that does CRUD (get, create, update, and delete) operations on a user profile in [mongoDB](https://www.mongodb.com/).
 - [User Profile with Gin-Mongo](/docs/go/quickstart/gin-mongo-2) - A sample URL Shortener app that responds with a shortened URL of the URL sent as a request. The application tests Keploy integration capabilities using [Gin](https://gin-gonic.com) and [mongoDB](https://www.mongodb.com/).
+- [Contact Keeper with Gorilla/Mux-Redis](/docs/go/quickstart/gorillamux-redis): A sample contact keeper app to test Keploy integration capabilities using Gorilla/Mux and Redis.
```
Comment (SanskritiHarmukh) [inline docs/go/quickstart/quickstart.md:17]: @developer-diganta can you add a one line detail of does the app does, like I can see it stores the contact of the user.

**Comment by SanskritiHarmukh** [comment]
Rest of the changes looks good to me.

**Comment by developer-diganta** [comment]
@SanskritiHarmukh Can you check once?

**Review by SanskritiHarmukh — APPROVED** [review]

File: src/components/GSoC.js
Comment (nehagup) [inline src/components/GSoC.js:1]: How is it related to this PR @developer-diganta ?

File: static/img/GSoC.png
Comment (nehagup) [inline static/img/GSoC.png:1]: How is it related to this PR @developer-diganta ?

**Comment by nehagup** [comment]
@developer-diganta please resolve conflicts.

**Comment by developer-diganta** [comment]
@nehagup made a new PR for this [here](https://github.com/keploy/docs/pull/165). Had by mistake discarded the previous commits for this pr.

**Pull request closed without merging**